api-docs/README.md

```diff
@@ -31,6 +31,7 @@ To access similar data in CSV format within Tock, please visit the
 - [Projects](projects.md): Fetches information about various projects.
 - [Reporting period (audit specific time period)](reporting-period-audit-specific.md): Fetches a list of all users who have not submitted a timecard for the specified period.
 - [Reporting periods](reporting-period-audit.md): Fetches a list of all available reporting periods and basic information about them.
+- [Submissions by range](submissions.md): Fetches a list of users and a count of punctual timecard submission by past periods
```

Comment:
"and a count of timecards submitted on time by past reporting period?"

Reply:
I was thinking it would be helpful for folks to know what Tock itself is considering as `on-time` or `punctual` here. Even just a plain-language description of this implementation would be helpful as I don't think there's an existing and consistently shared understanding of a date/time by which a timecard must be submitted.

Reply:
I added a bit to clarify when a timecard is considered "on time" as added in this PR (if my assumptions are wrong then we should change some code!)